pages/blog/index.vue

```diff
@@ -43,11 +43,20 @@
             {{ blogCategory }}
           </h2>
         </div>
-        <div>
+
+
+        <div class="search-input-container">
+          <span class="search-icon ">
+            <img
+              src="~/assets/icons/iconSearch.svg"
+              alt="Search Icon"
+            >
+          </span>
           <input
             v-model="searchQuery"
             type="text"
             placeholder="Search"
+            class="search-input"
             @keyup.enter="fetchBlogEntries()"
```

Comment:
When the input is clear or as you delete text, it should automatically clear the search. A user should not have to click on the search icon or click enter to see all posts again